### PR TITLE
update speakers-company-logo

### PR DESCRIPTION
**Pull request by rnrnshn**
updated the company logo below the speaker's photo to gdgmaputo logo as said in [Update speakers info #8 ](https://github.com/mozdevz/DevFest18/issues/8)

screenshot_12

### CODE REVIEW

**Comment by osvaldoM** [comment]
@rosariopfernandes can you test if this is working ? I'm unable to test it at the moment.

File: docs/default-firebase-data.json [hunk 1]
```diff
@@ -528,8 +528,8 @@
     "elton_laice" : {
       "bio" : "A 23 year-old youth, technology entrepreneur, full-stack developer and co-founder of kurrula-ia, codelaice, voluntariado.co.mz.",
       "company" : "Kurrula-ia, Codelaice",
-      "companyLogo" : "/images/logos/gdg-lviv.svg",
-      "companyLogoUrl" : "https://storage.googleapis.com/hoverboard-experimental.appspot.com/images/logos/gdg-lviv.svg",
+      "companyLogo" : "/images/logo.png",
+      "companyLogoUrl" : "images/logo.png",
```
Comment (thatfiredev) [inline docs/default-firebase-data.json]:
```suggestion
      "companyLogoUrl" : "https://firebasestorage.googleapis.com/v0/b/devfestmaputo.appspot.com/o/logo.png?alt=media&token=516663cb-88d6-45a5-a98d-eab5409dea58",
```

File: docs/default-firebase-data.json [hunk 2]
```diff
@@ -547,8 +547,8 @@
     "igor_sambo" : {
       "bio" : "Estudante de Engenharia Informática e de Telecomunicações, 3o ano no Instituto Superior de Transportes e Comunicação. Gosta de Programar e pretende ser expert arm Flutter e já apresentou um Study Jam sobre o assunto e continua fazendo pesquisas acerca.",
       "company" : "ISUTC",
-      "companyLogo" : "/images/logos/gdg-lviv.svg",
-      "companyLogoUrl" : "https://storage.googleapis.com/hoverboard-experimental.appspot.com/images/logos/gdg-lviv.svg",
+      "companyLogo" : "/images/logo.png",
+      "companyLogoUrl" : "images/logo.png",
```
Comment (thatfiredev) [inline docs/default-firebase-data.json]:
```suggestion
      "companyLogoUrl" : "https://firebasestorage.googleapis.com/v0/b/devfestmaputo.appspot.com/o/logo.png?alt=media&token=516663cb-88d6-45a5-a98d-eab5409dea58",
```

File: docs/default-firebase-data.json [hunk 3]
```diff
@@ -570,8 +570,8 @@
     "charmilo_narane" : {
       "bio" : "Charmilo é apenas um dos vários entusiastas da tecnologia que acreditam que com apenas um computador e uma conexão à internet as possibilidades e oportunidades para um mundo em constante crescimento são ilimitadas. Suas principais áreas de interesse são desenvolvimento, contabilidade e segurança cibernética.",
       "company" : "Universidade Wutivi-Unitiva, GreatTech Lda",
-      "companyLogo" : "/images/logos/gdg-lviv.svg",
-      "companyLogoUrl" : "https://storage.googleapis.com/hoverboard-experimental.appspot.com/images/logos/gdg-lviv.svg",
+      "companyLogo" : "/images/logo.png",
+      "companyLogoUrl" : "images/logo.png",
```
Comment (thatfiredev) [inline docs/default-firebase-data.json]:
```suggestion
      "companyLogoUrl" : "https://firebasestorage.googleapis.com/v0/b/devfestmaputo.appspot.com/o/logo.png?alt=media&token=516663cb-88d6-45a5-a98d-eab5409dea58",
```

File: docs/default-firebase-data.json [hunk 4]
```diff
@@ -594,7 +594,7 @@
       "bio" : "Auto-entitulado \"Melhor Web Developer de Moz\", Elísio Leonardo é estudante de Informática na UEM, criador do maior sistema integrado de gestão académica de Moçambique e do maior portal de Informática em Moçambique, www.infomoz.net",
       "company" : "Universidade Eduardo Mondlane",
       "companyLogo" : "/images/logo.png",
-      "companyLogoUrl" : "https://storage.googleapis.com/hoverboard-experimental.appspot.com/images/logos/gdg-lviv.svg",
+      "companyLogoUrl" : "/images/logo.png",
```
Comment (thatfiredev) [inline docs/default-firebase-data.json]:
```suggestion
      "companyLogoUrl" : "https://firebasestorage.googleapis.com/v0/b/devfestmaputo.appspot.com/o/logo.png?alt=media&token=516663cb-88d6-45a5-a98d-eab5409dea58",
```

File: docs/default-firebase-data.json [hunk 5]
```diff
@@ -612,8 +612,8 @@
     "edilson_ngulele" : {
       "bio" : "Edilson is an Android developer by passion. Enjoys coding, writing and talking about code. If he's not doing one of these, he's certainly reading about code or self-awareness.",
       "company" : "ISUTC",
-      "companyLogo" : "/images/logos/gdg-lviv.svg",
-      "companyLogoUrl" : "https://storage.googleapis.com/hoverboard-experimental.appspot.com/images/logos/gdg-lviv.svg",
+      "companyLogo" : "/images/logo.png",
+      "companyLogoUrl" : "/images/logo.png",
```
Comment (thatfiredev) [inline docs/default-firebase-data.json]:
```suggestion
      "companyLogoUrl" : "https://firebasestorage.googleapis.com/v0/b/devfestmaputo.appspot.com/o/logo.png?alt=media&token=516663cb-88d6-45a5-a98d-eab5409dea58",
```

File: docs/default-firebase-data.json [hunk 6]
```diff
@@ -631,8 +631,8 @@
     "richaldo_elias" : {
       "bio" : "Estudante de Engenharia Informática, que tem sonho de resolver problemas do mundo, e para que isso seja possível, acha que homens e máquinas trabalhando juntos seria mais rápido encontrar respostas para os problemas da humanidade e até descobrir problemas que não achamos que não temos. Completou o curso de Machine Learning da Standford University e Neural Network and Deep Learning da deeplearning.ai .",
       "company" : "Universidade Eduardo Mondlane",
-      "companyLogo" : "/images/logos/gdg-lviv.svg",
-      "companyLogoUrl" : "https://storage.googleapis.com/hoverboard-experimental.appspot.com/images/logos/gdg-lviv.svg",
+      "companyLogo" : "/images/logo.png",
+      "companyLogoUrl" : "images/logo.png",
```
Comment (thatfiredev) [inline docs/default-firebase-data.json]:
```suggestion
      "companyLogoUrl" : "https://firebasestorage.googleapis.com/v0/b/devfestmaputo.appspot.com/o/logo.png?alt=media&token=516663cb-88d6-45a5-a98d-eab5409dea58",
```

File: docs/default-firebase-data.json [hunk 7]
```diff
@@ -704,8 +704,8 @@
     "losdelau" : {
       "bio" : "Formado em Sistemas Informáticos pelo Instituto de Transportes e Comunicações. Estudante do curso de Licenciatura em Informática no Departamento de Matemática e Informática da Universidade Eduardo Mondlane.",
       "company" : "EXI, LDA",
-      "companyLogo" : "",
-      "companyLogoUrl" : "",
+      "companyLogo" : "/images/logo.png",
+      "companyLogoUrl" : "/images/logo.png",
```
Comment (thatfiredev) [inline docs/default-firebase-data.json]:
```suggestion
      "companyLogoUrl" : "https://firebasestorage.googleapis.com/v0/b/devfestmaputo.appspot.com/o/logo.png?alt=media&token=516663cb-88d6-45a5-a98d-eab5409dea58",
```

**Review by thatfiredev — CHANGES_REQUESTED** [review]
The field `companyLogoUrl` should contain a full URL and not a path to the image. I don't know the actual reason why, but that's how the template was structured when we got it, so I suggest we keep it that way:

File: docs/default-firebase-data.json (hunk 2 above)
Comment (rnrnshn) [reply docs/default-firebase-data.json]: Thanks That's the correct url

**Review by rnrnshn — COMMENTED** [review]
Thanks @rosariopfernandes I was looking for this url

**Review by thatfiredev — APPROVED** [review]
LGTM! I actually have some changes to add to the list of speakers, so I'll only deploy this once my changes are pushed.

**Comment by thatfiredev** [comment]
@0limpi0 on a side note: GitHub now supports batching suggested changes in a single commit (beta). See [this tweet](https://twitter.com/github/status/1062401381082968064)

**Pull request merged**